src/routes/index.js

```diff
@@ -2,11 +2,13 @@ import express from 'express';
 import authRoute from './auth.route';
 import tripRoute from './trip.route';
 import profileSettingsRoute from './profile-settings.route';
+import notificationRoute from './notification.route';
 
 const app = express();
 
 app.use('/api/auth', authRoute);
 app.use('/api/', tripRoute);
 app.use('/api/', profileSettingsRoute);
+app.use('/api/', notificationRoute);
```

Comment:
The context can be scoped to `/api/notifications`

Reply:
Yeah, it is /api/notifications except that the last part `/notification` is in the notification route file.